plume-models/src/blogs.rs

```diff
@@ -220,12 +225,46 @@ impl Blog {
         coll.collection_props.items = serde_json::to_value(self.get_activities(conn)?)?;
         coll.collection_props
             .set_total_items_u64(self.get_activities(conn)?.len() as u64)?;
+        coll.collection_props
+            .set_first_link(Id::new(ap_url(&format!("{}?page=0", &self.outbox_url))))?;
+        coll.collection_props
+            .set_last_link(Id::new(ap_url(&format!(
+                "{}?page={}",
+                &self.outbox_url,
+                self.get_activities(conn)?.len() as u64 / ITEMS_PER_PAGE as u64
```

Comment:
I think this does not give the right result, divisions are flooring where in this case we need them ceiling. An easy way to achive that is to add ITEMS_PER_PAGE-1 before dividing
```suggestion
                (self.get_activities(conn)?.len() + ITEMS_PER_PAGE - 1) as u64 / ITEMS_PER_PAGE as u64
```

Reply:
Alright, that makes sense too.